From 208008c322fd1ffdd07e7fc95910154174ce0ff3 Mon Sep 17 00:00:00 2001
From: Tomas Winkler <tomas.winkler@intel.com>
Date: Mon, 10 Feb 2014 14:33:53 +0200
Subject: [char-misc-next 09/39] mei: fix compilation error with missing
 WATCHDOG_CORE

Kconfig is not transitive so INTEL_ME_TXE has to depend
on WATCHDOG_CORE as well

ERROR: "watchdog_unregister_device" [drivers/misc/mei/mei.ko] undefined!
ERROR: "watchdog_register_device" [drivers/misc/mei/mei.ko] undefined!

Reported-by: Fengguang Wu <fengguang.wu@intel.com>
Signed-off-by: Tomas Winkler <tomas.winkler@intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/misc/mei/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/drivers/misc/mei/Kconfig b/drivers/misc/mei/Kconfig
index 5881679..d23384d 100644
--- a/drivers/misc/mei/Kconfig
+++ b/drivers/misc/mei/Kconfig
@@ -37,7 +37,7 @@ config INTEL_MEI_ME
 config INTEL_MEI_TXE
 	tristate "Intel Trusted Execution Environment with ME Interface"
 	select INTEL_MEI
-	depends on X86 && PCI
+	depends on X86 && PCI && WATCHDOG_CORE
 	help
 	  MEI Support for Trusted Execution Environment device on Intel SoCs
 
-- 
1.9.0

